packages/babel-core/test/option-manager.js

```diff
@@ -94,8 +94,8 @@ describe("option-manager", () => {
       expect(() => {
         loadOptions({
           plugins: [[plugin, null]],
-        }).toThrow(/.plugins[0][1] must be an object, false, or undefined/);
-      });
+        });
+      }).toThrow(".plugins[0][1] must be an object, false, or undefined");
```

Comment:
`toThrow` is chained at wrong function. Unfortunately it will always pass and `eslint-plugin-jest` helps detecting such bugs.